utils/spdx/src/main/resources/simple-license-mapping.yml

```diff
@@ -22,34 +22,8 @@
 # Sort the entries below via IntelliJ's "Edit" -> "Sort Lines".
 ---
 
-# Ambiguous mappings (mapping reason not obvious without additional information)
-ALv2: Apache-2.0
-ASL: Apache-2.0
-Apache-style: Apache-2.0
-Apache: Apache-2.0
```

Comment:
commit: The title says `Check for...`

But the commit also removes a bunch of entries. I recall in the past, that we removed only some `BSD` related entry which caused huge amounts of issues popping up.

So, I believe this is critical and should be made more prominent by dedicating the commit title to it.

Reply:
I can tweak commit messages later if we agree on the general idea and I move this PR out of draft state.